2/contrib/s2i/run

```diff
@@ -226,12 +226,31 @@ if [[ -n "${INSTALL_PLUGINS:-}" ]]; then
   /usr/local/bin/install-plugins.sh "${plugins_file}"
 fi
 
+copy_plugins() {
+    echo "Copying $(ls /opt/openshift/plugins | wc -l) Jenkins plugins to ${JENKINS_HOME} ..."
+    cp -frL /opt/openshift/plugins/* ${JENKINS_HOME}/plugins/
```

Comment:
does this copy *everything* and will trigger unzip/reload of all plugins or just the changed ones ?

just to understand how big impact we make here.

Reply:
it will copy everything